.envrc

```diff
@@ -0,0 +1 @@
+dotenv
```

Comment:
.envrc definitely should be checked in. I do not see what is offensive about it.